internal/controller/controllers/local_cluster_import_controller.go

```diff
@@ -267,6 +280,40 @@ func (r *LocalClusterImportReconciler) ensureLocalClusterCRsDeleted(ctx context.
 		r.log.Errorf("could not delete local AgentClusterInstall due to error %s", err.Error())
 		return err
 	}
+	err = r.deleteInfraEnv(ctx, r.localClusterName, r.localClusterName)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		r.log.Errorf("could not delete local InfraEnv due to error %s", err.Error())
+		return err
+	}
+	return nil
+}
+
+func (r *LocalClusterImportReconciler) createOrUpdateInfraEnv(ctx context.Context, pullSecretRefName string) error {
+	infraEnv := aiv1beta1.InfraEnv{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      r.localClusterName,
+			Namespace: r.localClusterName,
+		},
+		Spec: aiv1beta1.InfraEnvSpec{
+			ClusterRef: &aiv1beta1.ClusterReference{
+				Name:      r.localClusterName,
+				Namespace: r.localClusterName,
+			},
+			PullSecretRef: &v1.LocalObjectReference{
+				Name: pullSecretRefName,
+			},
+		},
+	}
+	mutateFn := func() error {
+		return nil
+	}
+	createOrUpdateResult, err := controllerutil.CreateOrUpdate(ctx, r.client, &infraEnv, mutateFn)
+	if err != nil {
+		return errors.Wrap(err, "could not create cluster image set")
```

Comment:
This error is wrong, should not be referencing `cluster image set`

Reply:
Fixed now